client/bootstrap/daemon-up.sh

```diff
@@ -33,7 +33,7 @@ sudo docker run -d --rm \
     -p "$PORT":8081 \
     -v /var/run/docker.sock:/var/run/docker.sock \
     -v $HOME:/app/host \
-    -e DAEMON="true" \
+    -e INERTIA_DAEMON='true' \
```

Comment:
Just an observation - it looks like use of " " and ' ' is inconsistent in our shell scripts.

Reply:
That's a good point... I'm not actually sure how interchangeable they are 🤔 

Will look into it

Reply:
I think the way we have it now is fine: [see this explanation for single vs double quotes in bash](https://stackoverflow.com/questions/6697753/difference-between-single-and-double-quotes-in-bash)

Single for string literals, double for things like `"$MYVAR"`

Reply:
Cool, thanks for the link!